api/src/services/settings.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
   },
   /**
    * Process a request to either replace, overwrite or extend existing settings.
-   * If both replace and overwite are set, then it is assumed that only overwite
+   * If both replace and overwrite are set, then it is assumed that only overwrite
```

Comment:
Consider adding a `@returns` doc comment now that we're intentionally returning something.

Reply:
Done